tests/app/data/test_schema_validation.py

```diff
@@ -83,8 +75,6 @@ def validate_json_against_schema(file, json_to_validate, schema):
             return []
         except jsonschema.exceptions.ValidationError as e:
             return ["Schema Validation Error! File [{}] does not validate against schema. Error [{}]".format(file, e)]
-        except JSONDecodeError as e:
```

Comment:
I find that the output at the moment in time doesn't really allow me to resolve more than one schema at a time if it fails because it's too verbose